gesture/gesture.go

```diff
@@ -146,33 +149,34 @@ func (c *Click) Events(q event.Queue) []ClickEvent {
 		}
 		switch e.Type {
 		case pointer.Release:
-			if !c.pressed || c.pid != e.PointerID {
+			if c.pid != e.PointerID {
 				break
 			}
-			c.pressed = false
+			b := c.Buttons ^ e.Buttons // the released button
```

Comment:
Did you mean "buttons" in plural?

Use full comment sentences: // The released buttons.